binding.gyp

```diff
@@ -13,7 +13,21 @@
           ],
           'libraries': [
             'shlwapi.lib'
-          ]
+          ],
+          'msvs_settings': {
+            'VCCLCompilerTool': {
+              'ExceptionHandling': 1,
```

Comment:
`ExceptionHandling` is unrelated to this change, should be removed.

Reply:
removed